guides/developer/extending/new-stage.md

```diff
@@ -83,7 +84,7 @@ public class ChapStage implements StageDefinitionBuilder, CancellableStage {
 
 ## Task classes
 
-A task class must implement a [com.netflix.spinnaker.orca.Task](https://github.com/spinnaker/orca/blob/master/orca-core/src/main/java/com/netflix/spinnaker/orca/Task.java), 
+A task class must implement a [com.netflix.spinnaker.orca.Task](https://github.com/spinnaker/orca/blob/master/orca-core/src/main/java/com/netflix/spinnaker/orca/Task.java),
```

Comment:
Dead link, should be https://github.com/spinnaker/orca/blob/master/orca-api/src/main/java/com/netflix/spinnaker/orca/api/pipeline/Task.java

Reply:
Package path is off as well (no longer `com.netflix.spinnaker.orca.Task`)